src/pages/legal/terms.mdx

```diff
@@ -61,7 +61,7 @@ We take no responsibility and assume no liability for any User Content that you
 
 By posting or otherwise making available any User Content on or through the Service, you expressly grant, and you represent and warrant that you have all rights necessary to grant and hereby grant to PrairieLearn, Inc. a royalty-free, sublicensable, transferable, perpetual, irrevocable, non-exclusive, worldwide license, subject to the restrictions set forth below to reproduce, modify, publish, publicly display, make derivative works and otherwise use such User Content and your name, voice, and/or likeness as contained in your User Content, in whole or in part, and in any form, media or technology, whether now known or hereafter developed, for use in connection with the Service and our (and its successors’ and affiliates’) business, including without limitation for promoting and redistributing part or all of the Service (and derivative works thereof) in any media formats and through any media channels. Notwithstanding the foregoing, we will only use your User Content as you provide in your settings page.
 
-If you are a Student, you agree that we may make your User Content available to your Teacher(s) and School(s), for the purposes of improving the Service only and at all times on a confidential basis, may use your Educational Records solely on an aggregated, de-identified basis and will not share such information with third parties.
+If you are a Student, you agree that we may (i) make your User Content available to your Teacher(s) and School(s), and (ii) for the purposes of improving the Service only and at all times on a confidential basis, use your Educational Records solely on an aggregated, de-identified basis and will not share such information with third parties.
```

Comment:
I think this paragraph was intending to use the same (i)/(ii) format as the paragraph immediately below. Without it the grammar doesn't make complete sense.